### DISCUSSION TITLE
I accidentally used the computer's useragent and not mine and then changed will I be blocked?

### DISCUSSION THREAD

**Discussion by MODSQ1234** [masterking32/MasterHamsterKombatBot#18] [Q&A] [answered]
I accidentally used the computer's useragent and not mine and then changed will I be blocked?

**Comment by XShaan** [comment]
> I accidentally used the computer's useragent and not mine and then changed will I be blocked?

No worries, just change it to your user agent and everything will be good to go.

**Comment by masterking32** [comment, marked as answer]
Who knows!
